src/features/movies/MovieList/movieListSaga.js

```diff
@@ -1,4 +1,4 @@
-import { all, call, put, takeEvery, select } from "redux-saga/effects";
+import { all, call, put, takeEvery, select, delay } from "redux-saga/effects";
```

Comment:
we can remove the delay import now 😉

Reply:
Good shot! I appreciate your sharp eye. The delay import is indeed unnecessary. I'll remove it 🙂